CHANGELOG.rst

```diff
@@ -69,22 +69,22 @@ Individual components
 0.32.0
 ------
 
-Renku ``0.32.0`` introduces improvements in the KG services, enhancing KG overall performance. 
+Renku ``0.32.0`` introduces improvements in the  services, enhancing  overall performance. 
 
 User-Facing Changes
 ~~~~~~~~~~~~~~~~~~~
 
 **✨ Improvements**
 
-- 🔁 **KG**: ``updatedAt`` renamed to ``dateModified`` on the Project Details API (`#1582 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1582>`_).
-- 📖 **KG**: Improved performance of the Project Status API (`#1554 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1554>`_).
+- 🔁 ****: ``updatedAt`` renamed to ``dateModified`` on the Project Details API (`#1582 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1582>`_).
```

Comment:
I happened to revise the way we were naming the knowledge graph, and before Laura's leave, we were using Knowledge Graph instead of KG. Hence, I have changed the KG in the current changelog. You are right, I may have removed previous ones by mistake when doing the search. Thank you!

Reply:
Thank you!